os/board/rtl8730e/src/component/soc/amebad2/bootloader/bootloader_hp.c

```diff
@@ -280,7 +280,7 @@ void BOOT_DDR_Init(void)
 #ifdef CONFIG_LINUX_FW_EN
 	rxi316_DynSre_init(0xFFFF, ENABLE);		//for linux lcdc, set the max count
 #else
-	rxi316_DynSre_init(0x1FF, ENABLE);
+	rxi316_DynSre_init(0xFFF, ENABLE);
```

Comment:
No need to add REV06 conditional?

Reply:
BOOT_Image1() will perform the checking for the ChipInfo_MemoryType() 
[Memory_Type_PSRAM or Memory_Type_DDR]

line 659 else case (means Memory_Type_DDR) will perform the BOOT_DDR_Init.

Reply:
@ZhenBei-Sin Sorry I don't get your point. What I asked is that this change is working on previous and new packages all? Because this does not have REV06 conditional checking, I asked.

Reply:
For non-DDR packages (previous packages) it's REV05 and below, for new DDR packages (New) it's REV06 and above.
Thus, we need to only differentiate DDR and non-DDR from our driver's side.

Reply:
Hi @sunghan-chang , actually many of the codes are running on KM4 and KM0. But we have a same reference in TizenLite as well (ie. I mean CA32 doesn't run these codes).
Thus, the thing we have to take care is the code in KM4 and KM0 cores that run those code take changes respectively.
This part is to increase the DDR auto gating timeout (ie. DDR self refresh) from 0x1FF -> 0xFFF. As you know, DDR differs from PSRAM, DDR need to explicitly refresh the data to keep the data retained (ie. capacitor will have leakage over time).
Thus, we need to increase the timeout value here, or else when it work with LCDC, it would cause frame rendering error due to short refresh timeout (ie. DDR self refresh more frequent, thus impacting LCDC working operation)